api/all.md

```diff
@@ -10,14 +10,14 @@ includeTOC: true
   * [Library API](https://www.scala-lang.org/api/2.13.6/)
   * [Compiler API](https://www.scala-lang.org/api/2.13.6/scala-compiler/scala/)
   * [Reflection API](https://www.scala-lang.org/api/2.13.6/scala-reflect/scala/reflect/)
-* Scala 2.12.13
-  * [Library API](https://www.scala-lang.org/api/2.12.13/)
-  * [Compiler API](https://www.scala-lang.org/api/2.12.13/scala-compiler/scala/)
-  * [Reflection API](https://www.scala-lang.org/api/2.12.13/scala-reflect/scala/reflect/)
+* Scala 2.12.14
+  * [Library API](https://www.scala-lang.org/api/2.12.14/)
+  * [Compiler API](https://www.scala-lang.org/api/2.12.14/scala-compiler/scala/)
+  * [Reflection API](https://www.scala-lang.org/api/2.12.14/scala-reflect/scala/reflect/)
   * Scala Modules
-    * [XML API](https://www.scala-lang.org/api/2.12.13/scala-xml/scala/xml/)
-    * [Parser Combinators API](https://www.scala-lang.org/api/2.12.13/scala-parser-combinators/scala/util/parsing/)
-    * [Swing API](https://www.scala-lang.org/api/2.12.13/scala-swing/scala/swing/)
+    * [XML API](https://www.scala-lang.org/api/2.12.14/scala-xml/scala/xml/)
+    * [Parser Combinators API](https://www.scala-lang.org/api/2.12.14/scala-parser-combinators/scala/util/parsing/)
+    * [Swing API](https://www.scala-lang.org/api/2.12.14/scala-swing/scala/swing/)
```

Comment:
I think the 2.12.13 version should be copied down to the `## Previous releases` section